### ISSUE TITLE
Query for providers

### ISSUE THREAD

**Issue by bspaans** [ankyra/escape-inventory#11]
To make it easier to deploy packages that require providers to be present it would be nice if the Inventory could show which packages implement a specific interface:

`/api/v1/inventory/__providers?query=kubernetes`

Could return something like e.g.:

```
{
  "_/kubernetes-provider-v0.0.1": {
    "project": ...,
     ....
  }
}
```

This could be a huge payload though, so maybe it should (have the option to) only return the latest version of a package that implements that interface.

**Comment by bspaans** [comment]
Decided to only keep track of latest versions implementing a provider interface.

**Comment by bspaans** [comment]
Implemented in escape-inventory-v0.14.11. When upgrading providers for existing releases won't be processed.
